lib/phobos_db_checkpoint/handler.rb

```diff
@@ -31,7 +45,15 @@ def around_consume(payload, metadata)
           end
 
           event_action = instrument('db_checkpoint.event_action', event_metadata) do
-            yield
+            begin
```

Comment:
Mikael brought an interesting point. Maybe we don't have to default. =P

If the user needs to make a conscious choice about this property, then there will be no surprises. So, in the future, the consumer groups could break and not start at all if the config is not set.

For now, I agree with @klippx idea of: when the config is not set, default to the old behaviour (infinity retries) _but_ warns the user about the "deprecation" (or future requirement) of explicitly choosing:

```yaml
listeners:
  - handler: MyHandler
    topic: le_topic
    # and then either
    max_retries: 3 # the handler will retry this many times each message before storing them in the quarantine Failure table
    # or
    retry_forever: true
```